.github/workflows/cd.yml

```diff
@@ -0,0 +1,27 @@
+name: Deploy
```

Comment:
this is not a Deploy job; please look at the end of your workflow and put here the correct name here

Reply:
@Nkbtemmy, I wrote this for the deploy job, but we don't have any deployment environment ready.